Sources/WalletConnectKMS/Serialiser/Serializer.swift

```diff
@@ -59,15 +60,23 @@ public class Serializer: Serializing {
         }
     }
 
-    private func handleType1Envelope<T: Codable>(_ topic: String, peerPubKey: Data, sealbox: Data) throws -> (T, String) {
+    private func rawRequest(_ topic: String, _ envelope: Envelope) throws -> String? {
+        if let symmetricKey = kms.getSymmetricKeyRepresentable(for: topic) {
+            return try String(data: codec.decode(sealbox: envelope.sealbox, symmetricKey: symmetricKey), encoding: .utf8)
```

Comment:
why do you convert it into the string as later you hash rawRepresentation of that string?